atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CqlExecutorImpl.java

```diff
@@ -228,6 +231,7 @@ private static Arg<Long> limit(long limit) {
         return SafeArg.of("limit", limit);
     }
 
+    @Unsafe
     private static Arg<String> quotedTableName(TableReference tableRef) {
         String tableNameWithQuotes = "\"" + CassandraKeyValueServiceImpl.internalTableName(tableRef) + "\"";
```

Comment:
why do we want this tableName to be unsafe?

Reply:
Ah, this is an unfortunate consequence of the way the annotations work :( - I loosely quote @carterkozak, but the function of `@Unsafe` is to stop people from doing `SafeArg.of("name", unsafeAnnotatedThing)`. In particular, for `Arg`s themselves the annotation is pretty meaningless. Here and in places mentioned later I'll leave some comments.